datahub-web-react/src/app/entity/shared/tabs/Dataset/Queries/utils/filterQueries.ts

```diff
@@ -0,0 +1,18 @@
+import { Query } from '../types';
+
+/**
+ * Filter queries by a search string. Compares name, description, and query statement.
+ *
+ * @param filterText the search text
+ * @param queries the queries to filter
+ */
+export const filterQueries = (filterText, queries: Query[]) => {
```

Comment:
nit: could add a unit test for this pure fxn. but if we want to get this out feel free to ignore!